ddtrace.gemspec

```diff
@@ -28,7 +28,14 @@ Gem::Specification.new do |spec|
     raise 'RubyGems 2.0 or newer is required to protect against public gem pushes.'
   end
 
-  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.files =
+    `git ls-files -z`
+    .split("\x0")
+    .reject { |f| f.match(%r{^(test|spec|features|[.]circleci|[.]github|[.]dd-ci|benchmarks|gemfiles|integration|tasks)/}) }
+    .reject do |f|
+      ['.dockerignore', '.env', '.gitattributes', '.gitlab-ci.yml', '.rspec', '.rubocop.yml',
+       '.rubocop_todo.yml', '.simplecov', 'Appraisals', 'Gemfile', 'Rakefile', 'docker-compose.yml'].include?(f)
```

Comment:
What this point, what do you think of listing the included files instead?

Reply:
I'm not the biggest fan of that, since I've seen quite a few broken releases on rubygems.org due to "it worked on my machine but we forgot to update the files list and so the packaged gem is missing stuff".

On the other hand, those projects usually specified all files by name, and I think you may be suggesting to just specify the ones on the the root, which is a smaller subset, but... I'd still go with the safer default of excluding what we don't want.